camunda-sdk-java/java-common/src/main/java/io/camunda/common/auth/SimpleAuthentication.java

```diff
@@ -89,4 +88,10 @@ public Map.Entry<String, String> getTokenHeader(Product product) {
 
     return new AbstractMap.SimpleEntry<>("Cookie", token);
   }
+
+  @Override
+  public void resetToken(Product product) {
+    SimpleCredential simpleCredential = simpleConfig.getProduct(product);
+    retrieveToken(product, simpleCredential);
```

Comment:
Why not just remove the token from `tokens` to make sure an invalid token is removed?

Reply:
Addressed in https://github.com/camunda-community-hub/spring-zeebe/pull/588/commits/137ff5b93dc9b589063987d6edf39b9418cf9841